### PR TITLE
Add runbook for the KubeAPIErrorBudgetBurn alert

### PR DESCRIPTION
**Pull request by ihcsim**
This PR adds a runbook with steps to diagnose and mitigate the
`KubeAPIErrorBudgetBurn` alerts. There have been a number of [BZ
issues][bugzilla list] on these alerts but not much have been documented on what they
mean and how to handle them. This runbook is the first attempt to fill in the 
gaps.

Unfortunately, there isn't a straightforward way to get to the root cause of
these alerts. This runbook attempts to provide some relevant metrics and tools
that can be used to help get closer to the root cause. 

See this [BZ][bugzilla issue] for recent context.

[bugzilla list]: https://bugzilla.redhat.com/buglist.cgi?quicksearch=KubeAPIErrorBudgetBurn&list_id=11939813
[bugzilla issue]: https://bugzilla.redhat.com/show_bug.cgi?id=1953798

### CODE REVIEW

**Comment by RiRa12621** [comment]
/approve
@NautiluX can you take a look as well?

**Comment by RiRa12621** [comment]
/lgtm
Huge thanks for that!

File: alerts/KubeAPIErrorBudgetBurn.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+/ scalar(sum(rate(apiserver_request_total{job="apiserver",verb=~"LIST|GET"}[1d])))
+, "type", "error", "_none_", "")
+or
+# resource-scoped latency
+label_replace(
+  (
+    sum(rate(apiserver_request_duration_seconds_count{job="apiserver",verb=~"LIST|GET",subresource!~"proxy|log|exec",scope="resource"}[1d]))
+  -
+    (sum(rate(apiserver_request_duration_seconds_bucket{job="apiserver",verb=~"LIST|GET",subresource!~"proxy|log|exec",scope="resource",le="0.1"}[1d])) or vector(0))
+  ) / scalar(sum(rate(apiserver_request_total{job="apiserver",verb=~"LIST|GET",subresource!~"proxy|log|exec"}[1d])))
```
Comment (aojea) [inline alerts/KubeAPIErrorBudgetBurn.md:58]: this first right parethesis is wrong

Comment (aojea) [reply alerts/KubeAPIErrorBudgetBurn.md:58]: ah, no, sorry , is correct, I was missing the first one

**Pull request merged**